privacyidea/lib/passwordreset.py

```diff
@@ -145,7 +145,8 @@ def is_password_reset():
     policy_at_all = Policy.get_policies(scope=SCOPE.USER, active=True)
     log.debug("Policy at all: {0!s}".format(policy_at_all))
     policy_reset_pw = Policy.get_policies(scope=SCOPE.USER,
-                                          action=ACTION.PASSWORDRESET)
+                                          action=ACTION.PASSWORDRESET,
+                                          active=True)
```

Comment:
I'm not 100% sure I understand the logic here:
Doesn't ``policy_reset_pw == True`` automatically imply ``policy_at_all == True``?

Reply:
The logic is:

I there are not any user policies, the user is allowed to do everything - also reset the password.

If there is ANY password policy, the user is only allowed to reset the password, if there is a password reset policy.